src/components/Chips/Chips.tsx

```diff
@@ -75,12 +76,21 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
     ref
   ) => {
     const overrideDataTestId = dataTestId || getTestId(ComponentDefaultTestId.CHIP, id);
+    const [hoverRef, isHovered] = useHover();
     const componentRef = useRef(null);
-    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef, hoverRef] });
 
     const backgroundColorStyle = useMemo(() => {
-      return { backgroundColor: disabled ? getCSSVar("disabled-background-color") : getElementColor(color, true) };
-    }, [disabled, color]);
+      let cssVar;
+      if (disabled) {
+        cssVar = getCSSVar("disabled-background-color");
+      } else if (isHovered) {
```

Comment:
according to the design, hover state should be added only when the chip is clickable. This should happen only if there is a onMouseDown prop or onClick (which is not exist right now and we need to add)